### ISSUE TITLE
ListView does not redraw on insertion

### ISSUE THREAD

**Issue by Guiguiprim** [slint-ui/slint#3125]
With `slint 1.1.1` on Windows in rust.

With a `VecModel<MyData>` in a `ListView` when inserting (with `model.insert(0, data)`) the view does not show the new entry until a redraw is triggered (by a mouse move).

As a workaround calling `handle.window().request_redraw();` after the insertion works.

**Comment by ogoffart** [comment]
I can reproduce by adding this code in main.rs of the todo example

```rust
    let m = todo_model.clone();
    slint::Timer::single_shot(std::time::Duration::from_millis(2000), move || {
        println!("Ok.");
        m.insert(0, TodoItem { checked: true, title: "Timer happened".into() });
    });
```

**Comment by martgoodall** [comment]
I still have this issue on version 1.14.1, 1.15.1 and pre-release/1.16. My issue is on .remove but its the same overall lack of redraw. Have the display scrolled to the very bottom item, and get the code to remove an item that is higher up the list. The list only updates with a mouse scroll, or individual updates by clicking on each line. I cant provide an example but believe the above code would achieve the same. Please reopen ticket.
